packages/harpguru-core/src/components/hole-number/hole-number.test.tsx

```diff
@@ -2,15 +2,17 @@ import { useGlobal } from 'reactn'
 import React from 'react'
 import { render } from '@testing-library/react-native'
 
-import { activeCellsHarpStrata } from '../../test-resources'
-
 import { HoleNumber, HoleNumberIds } from './hole-number'
 
 jest.mock('reactn')
 const mockUseGlobal = useGlobal as jest.Mock
+// TODO: Should be shared resource
```

Comment:
Already addressed in a code review above, but this comment needs removing and the code modifying here too.